python_ga/genetic_algorithm.py

```diff
@@ -300,306 +40,24 @@ def mutations(population, config, iteration):
         'mutate_swap':         mutate_swap,
         'mutate_dose_value':   mutate_dose_value,
         'mutate_time_value':   mutate_time_value,
-        'mutate_merge':     mutate_merge,
         'mutate_split':     mutate_split,
     }
 
     for mut_type in list(config['mutations'].keys()):
-        config['mutations'][mut_type]['mut_prob'] = calculate_probability_annealing(iteration)
-        population = mutation[mut_type](population=population, config=config)
-        logger.info(f'{mut_type} {[sum(pop) for pop in population]}')
+        if mut_type in mutation.keys():
```

Comment:
adding condition if all mutation types are implemented:

why:
`mutate_merge` removed (same as `mutate_time_value`), this `if` allows to use older configs without change